.env.example

```
@@ -7,6 +7,7 @@ SPREE_PUBLISHABLE_KEY=your_publishable_api_key
 # Set these to your store's default_country_iso and default_locale
 NEXT_PUBLIC_DEFAULT_COUNTRY=us
 NEXT_PUBLIC_DEFAULT_LOCALE=en
+NEXT_PUBLIC_STORE_NAME=Spree Store
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)